extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -1089,6 +1089,13 @@ export class SchemaCompareMainWindow {
 		}
 
 		let fileUri = fileUris[0];
+		this.openScmpFile(fileUri, true);
+
+	}
+	public async openScmpFile(fileUri: vscode.Uri, callFromWithinSC: boolean = false): Promise<void> {
+		if (!callFromWithinSC) {
+			await this.launch(undefined, undefined, false, undefined);
```

Comment:
does this open a new schema compare window?

Reply:
Yes, this does open a new SC window. And later in following lines, it reads the file and populates the source and target information appropriately.

Reply:
ok cool can you add a comment about that and why it's needed now that there's this new entry point?

Reply:
Sure, will do.